gradle/verification-metadata.xml

```diff
@@ -0,0 +1,111 @@
+         <trusted-key id="cc4483cd6a3eb2939b948667a1b4460d8ba7b9af" group="org.mockito" name="mockito-core" version="1.9.5"/>
+         <trusted-key id="d4c89ea4aaf455fd88b22087efe8086f9e93774e" group="junit" name="junit" version="4.12"/>
+         <trusted-key id="fa7929f83ad44c4590f6cc6815c71c0a4e0b8edd" group="net.java.dev.jna"/>
+      </trusted-keys>
+      <trusted-artifacts>
+         <!-- See why adding following trusts here:
+          https://docs.gradle.org/7.4.2/userguide/dependency_verification.html#sec:skipping-javadocs -->
+         <trust file=".*-javadoc[.]jar" regex="true"/>
+         <trust file=".*-sources[.]jar" regex="true"/>
+      </trusted-artifacts>
+   </configuration>
+   <components>
+      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.8.1">
+         <artifact name="gradle-enterprise-gradle-plugin-3.8.1.jar">
+            <sha512 value="11bed06d6ecd5669de3f32a0b386a223c2b2ba532a3c80d3e1613b29a1a264e1a75cdf142273f17ee1845287c76cf1f9d68cd97542318b1a9126104ce958824a" origin="Generated by Gradle because a key couldn't be downloaded"/>
+         </artifact>
+         <artifact name="gradle-enterprise-gradle-plugin-3.8.1.pom">
+            <sha512 value="bf282ac6b7c7f0b1728ef91581084f418c898c48bf83f43b2cb4549bf3cacf2e69551bbf904039a42dc536934230fcb9cf6826162d6cae1e08c203a183a2516e" origin="Generated by Gradle because artifact wasn't signed"/>
+         </artifact>
+      </component>
+      <component group="junit" name="junit" version="4.12">
+         <artifact name="junit-4.12.jar">
+            <sha512 value="5974670c3d178a12da5929ba5dd9b4f5ff461bdc1b92618c2c36d53e88650df7adbf3c1684017bb082b477cb8f40f15dcf7526f06f06183f93118ba9ebeaccce" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-4.12.pom">
+            <sha512 value="bad5814d39ed0eccbab35272b004de4109bfba2bf96f8cec262f02d58a810602378c6eb9f4d4587afcc5573141faad5936c914e2217f1fc798b4396e375aeb1b" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="net.java.dev.jna" name="jna" version="4.5.2">
+         <artifact name="jna-4.5.2.jar">
+            <sha512 value="41d3b175af8aca64dced867f137149019d3a2c0e45eacb84c3a7d066b1fcb66de154c0d0739835df22bf6609f7e779e017878dd27452c59213368cf73fb83f2d" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="jna-4.5.2.pom">
+            <sha512 value="7221025ec583a451318883d349c1e670e4f70332f1c871caece8843e267b185b426ee9905dc3c0146ef47c2c5256f5321b0885354b5501aa49d6f8309268a51d" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="net.java.dev.jna" name="jna-platform" version="4.5.2">
+         <artifact name="jna-platform-4.5.2.jar">
+            <sha512 value="440e81a7c7ded403c3190d81dc372a6559ffc4f9cb13d7d07328f25c34887f2df594652aeb04308ded8b473a5422ef45c58c7d6d2677ce541372fb52afc7a792" origin="Generated by Gradle"/>
```

Comment:
This cannot be matched with the dependencies.properties, which is the actual source of truth.

Can Gradle include the sha256 hash instead?

dependencies.properties includes this about jna-platform:

```
jna-platform.filename=jna-platform-4.5.2.jar
jna-platform.sha256=f1d00c167d8921c6e23c626ef9f1c3ae0be473c95c68ffa012bc7ae55a87e2d6
```

We need an entry that connects the gradle-hash to the dependencies.properties hash to be able to switch.